client/bower_components/app-localize-behavior/app-localize-behavior.html

```diff
@@ -291,7 +291,7 @@
           translatedValue = that.__findTranslatedValue(resources, fallbackLanguage, key)
       }
 
-      if (!translatedValue) {
+      if (!translatedValue && translatedValue !== "") {
```

Comment:
Did you yourself not tell me to put parenthesis around `(translatedValue !== "")` :-)